requirements.txt

```diff
@@ -26,7 +26,7 @@ coverage==5.4
 factory-boy==3.2.0
 django-anymail==8.2
 django-test-plus==2.2.0
-django-crontab==0.7.1
+apscheduler
```

Comment:
pin it?

Reply:
just added the version number to apscheduler